README.md

```diff
@@ -257,13 +256,14 @@ end
 ```
 
 * This ensures that if your app is using JWT tokens, cookies are no longer used
```

Comment:
`This ensures that if your app is using JWT app, the shop domain can be validated appropriately without cookies` ?